### DISCUSSION TITLE
endpoint returns "resource not found" error message

### DISCUSSION THREAD

**Discussion by depo101** [etsy/open-api#916] [Q&A] [answered]
Hello everyone, I'm working on a e-commerce integrations system to etsy.com. So I try to get the listings of my customer. 
I'm using open api v3. I passed the 3 steps below succeeded

- pinging
- authenticating
- getting access token

but when I try to access the example end point first time, I get the error message below
**"resource not found"**

My request parameters are below
**URL :**  https://api.etsy.com/v3/application/listings?state=draft or active
**authorization :** Bearer xxx.yyy
**x-api-key :** zzz

I did this request via Postman App on my local machine. 
I did the request as a "get" request. Also tried "post" request and the same error I did.

Best regards,

**Comment by PMStanley** [comment, marked as answer]
That endpoint doesn't exist, if you're trying to get all the active listings for a shop use this endpoint: 
https://openapi.etsy.com/v3/application/shops/{shop_id}/listings

API Spec: https://developer.etsy.com/documentation/reference/#operation/getListingsByShop

**Reply by depo101** [reply]
thanks for reply. I fix the url address and I got the list of products. But I could not find the right resource for getting the shop_id. Could you please share any resource how to we can get the shop_id. I got my shop id via this tool "https://app.cartrover.com/get_etsy_shop_id.php" Thanks.

**Reply by PMStanley** [reply]
> thanks for reply. I fix the url address and I got the list of products. But I could not find the right resource for getting the shop_id. Could you please share any resource how to we can get the shop_id. I got my shop id via this tool "https://app.cartrover.com/get_etsy_shop_id.php" Thanks.

I strongly suggest you spend some time looking through the API spec I linked, everything you need is there.  The shop ID can be found several ways but the easiest is the new getMe endpoint: https://developers.etsy.com/documentation/reference#operation/getMe

**Reply by depo101** [reply]
Thank you very much. I started to learn from "guides". Now I switched to "api reference". Thank you. Best regards.
